### PR TITLE
Remove CT related classes and usage

### PR DESCRIPTION
**Pull request by om-sharma**

### CODE REVIEW

File: src/test/java/uk/gov/functional/CleanDatabaseRule.java [hunk 1]
```diff
@@ -1,15 +1,10 @@
...
 import org.junit.rules.ExternalResource;
-import uk.gov.functional.db.TestEntriesDAO;
 
-public class CleanDatabaseRule extends ExternalResource {
-    public final TestEntriesDAO testEntriesDAO;
-
-    public CleanDatabaseRule(TestEntriesDAO testEntriesDAO) {
-        this.testEntriesDAO = testEntriesDAO;
-    }
+import static uk.gov.functional.DBHandle.*;
```
Comment (philandstuff) [inline src/test/java/uk/gov/functional/CleanDatabaseRule.java]: what's going on here? Is this related to the CT stuff or should it be a separate commit with its own explanation?

**Comment by philandstuff** [comment]
:+1: other than the commit above :point_up:

**Comment by om-sharma** [comment]
Reverted!

File: src/test/java/uk/gov/functional/CleanDatabaseRule.java (hunk 1 above)
Comment (om-sharma) [inline src/test/java/uk/gov/functional/CleanDatabaseRule.java]: Reverted these specific changes, will do in new PR.

**Comment by philandstuff** [comment]
can you squash the commits together? `git rebase -i master` and then the `fixup` option
